msteams-platform/tabs/how-to/create-personal-tab.md

```diff
@@ -994,6 +994,67 @@ If you create a bot with a **personal** scope, it appears in the first tab posit
 
 ```
 
+## Extend static tabs to group chat and meetings
+
+> [!NOTE]
+> * Extend your static tabs to group chat and meetings is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+> * To extend your static tabs to group chat and meetings, use the app manifest v1.16 or later.
+
+You can extend static tabs to group chat and meetings. Instead of pinned app content, you can build tabs that behave more like apps as you can pin only one tab per app for example, pinning a single YouTube app tab.
+
+To extend your static tabs to group chat and meetings, update your [app manifest](~/resources/schema/manifest-schema.md#statictabs) with the `scopes` and `context` parameters in the `staticTabs` property.
+
+Following is an example of app manifest where a static tab is defined that works in all scopes and contexts in Teams:
+
+```json
+"staticTabs": [ 
+  { 
+     "entityId": "homeTab", 
+     "scopes": [ 
+       "personal", 
+       "groupChat" 
+      ], 
+     "context": [ 
+       "personalTab",  
+       "privateChatTab", 
+       "meetingChatTab", 
+       "meetingDetailsTab", 
+       "meetingSidePanel", 
+       "meetingStage" 
+      ], 
+      "name": "Contoso", 
+      "contentUrl": "https://contoso.com/content (displayed in Teams canvas)", 
+      "websiteUrl": "https://contoso.com/content (displayed in web browser)" 
+  }
+], 
+
+```
+
+If a context isn't defined in the app manifest, by default Teams consider the following context:
+
+```json
+"context": [ 
+   "personalTab", 
+   "privateChatTab", 
+   "meetingChatTab", 
+   "meetingDetailsTab", 
+   "meetingStage" 
+]
+```
+
+## Customizing your static tab in chats or meetings
+
+To customize your static tab experience in chats or meetings, you can add `setConfig` APIs in a [configuration dialog](~/tabs/how-to/create-tab-pages/configuration-page.md#configuration-page-for-tabs) to update the `contentUrl`. Following is an example:
```

Comment:
We shouldn't say "in a configuration dialog". That's incorrect. Should say something like "you can use the setConfig APIs in your tab to update the contentUrl and/or websiteUrl.